readme.md

```diff
@@ -69,11 +69,18 @@ gulp.task('default', () => {
 
 ##### timeout
 
-Type: `number`  
+Type: `number`
 Default `5000`
 
 Time to wait in milliseconds before a test automatically fails.
 
+##### noFail
```

Comment:
Yeah, I'm not a fan of the double negation either. What do you think of errorOnFail (default: true)?